rbx_binary/src/core.rs

```diff
@@ -1,17 +1,41 @@
-use std::{
-    io::{self, Read, Write},
-    mem,
-};
+use std::io::{self, Read, Write};
 
 use rbx_dom_weak::Ustr;
 use rbx_reflection::{
     ClassDescriptor, PropertyDescriptor, PropertyKind, PropertySerialization, ReflectionDatabase,
 };
 
+use crate::chunk::ChunkBuilder;
+
 pub static FILE_MAGIC_HEADER: &[u8] = b"<roblox!";
 pub static FILE_SIGNATURE: &[u8] = b"\x89\xff\x0d\x0a\x1a\x0a";
 pub const FILE_VERSION: u16 = 0;
 
+pub struct ReadInterleavedBufferIter<const N: usize> {
+    buffer: Vec<u8>,
+    index: usize,
+    len: usize,
+}
+impl<const N: usize> ReadInterleavedBufferIter<N> {
```

Comment:
```suggestion
}

impl<const N: usize> ReadInterleavedBufferIter<N> {
```

Would rather we have new lines between blocks like this.